pushjack/apns.py

```diff
@@ -513,11 +509,7 @@ def to_json(self):
 
     def __len__(self):
         """Return length of serialized message."""
-        if not self.to_dict():
-            # Consider message length 0 if there is no message data.
-            return 0
-        else:
-            return len(self.to_json())
+        return len(self.to_json())
```

Comment:
Because `aps` is not removed if empty anymore above - the `self.to_dict` should always return at least `{"aps": {}}` - which would never be a `0` length string in JSON.